unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -127,3 +178,27 @@ def execute_query(
                 schema=schema,
                 table_name=table,
             ) from e
+
+    def get_information_schema(self, table_name: str) -> dict[str, str]:
+        """Function to generate information schema of the big query table.
+
+        Args:
+            table_name (str): _description_
+
+        Returns:
+            dict[str, str]: _description_
+        """
+        bigquery_table_name = str.lower(table_name).split(".")
```

Comment:
@kirtimanmishrazipstack is it guaranteed that we'll get an array of size 3? The user can enter any string right? How do we handle the errors in that scenario?

Reply:
It would be better if we include a `len` check here with `BIG_QUERY_TABLE_SIZE`